### PR TITLE
Inject SVG icons before rendering preview in Netlify CMS

### PR DESCRIPTION
**Pull request by contolini**
A little hacky but it works and I'm unsure of a better way. I added a browser test to catch future regressions.

## Changes

- The variation template strips out any icon include statements before rendering.

## Testing

1. Cloud browser tests should pass below.
1. `yarn test:browser` should pass locally.
1. Editing the buttons pages on the [PR preview](https://deploy-preview-493--cfpb-design-system.netlify.app/design-system/admin/#/collections/components/entries/buttons) should correctly show the preview pane while on [production](https://cfpb.github.io/design-system/admin/#/collections/components/entries/buttons) it throws an error and is blank.

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: docs/admin/src/widgets/variationPreviewTemplate.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import { Tabs } from 'govuk-frontend';
 import marked from 'marked';
 import template from '../../../_includes/variation-content.html';
 
+// react-liquid (https://github.com/aquibm/react-liquid/) isn't able to `include` other files so we
+// strip out any instances of {% include icons/XXXXX.svg %}
+const cleanTemplate = template.replace( /{%\s+include\s+\/?icons\/([\w-]+)\.svg\s+%}/g, '' );
```
Comment (anselmbradford) [inline docs/admin/src/widgets/variationPreviewTemplate.js:10]: If you're doing a find and replace, why don't we find and replace the icon file name with the icon SVG?

**Comment by Scotchester** [comment]
I'm not 100% sure I understand the problem this PR is trying to solve, but our preferred method of including SVGs is having their raw SVG code be present in the final page markup. Can we do that here instead of the `img src` method?

File: docs/admin/src/widgets/variationPreviewTemplate.js (hunk 1 above)
Comment (contolini) [reply docs/admin/src/widgets/variationPreviewTemplate.js]: Good thinking! I pointed them to the SVG files.

**Comment by contolini** [comment]
> I'm not 100% sure I understand the problem this PR is trying to solve, but our preferred method of including SVGs is having their raw SVG code be present in the final page markup. Can we do that here instead of the `img src` method?

This fix is just for the Netlify CMS preview pane that has to convert our liquid templates into React components. The final page markup is compiled via Jekyll and has the SVGs inlined.

**Comment by Scotchester** [comment]
> This fix is just for the Netlify CMS preview pane that has to convert our liquid templates into React components. The final page markup is compiled via Jekyll and has the SVGs inlined.

Cool, all good with me, then.

**Comment by contolini** [comment]
Update: @anselmbradford had the 🔥 idea of using webpack to inject the SVGs so now we're doing that instead of `img src`.

Note: `{% include icons/XXXXX.svg %}` statements that users add to a form field as part of a page's content, like in the restrictions section, do NOT get transpiled. This is because the preview pane is rendered on every keystroke and it'd be really expensive (thus, laggy) if we iterated through every field and swapped in SVGs in real time as people typed. Instead, in this PR, we just do it once when we load the static template file.

**Pull request merged**